packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -54,9 +56,13 @@ export const useAuthModal = props => {
     }, [showMyAccount]);
 
     const handleSignOut = useCallback(() => {
+        // After logout, reset the store to set the bearer token.
+        // https://www.apollographql.com/docs/react/networking/authentication/#reset-store-on-logout
+        apolloClient.resetStore();
+
         // TODO: Get history from router context when implemented.
         signOut({ history: window.history });
-    }, [signOut]);
+    }, [apolloClient, signOut]);
```

Comment:
I would expect `apolloClient` to be a stable object, but just in case, I would consider destructuring `resetStore` and using it as the dependency.

```es6
const { resetStore } = apolloClient

const handleSignOut = useCallback(
  () => {
    resetStore()
  },
  [resetStore]
)
```